### PR TITLE
Add almighty Bronzong

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: mods/essb/abilities.js [hunk 1]
```diff
@@ -698,4 +698,37 @@ exports.BattleAbilities = {
...
+	"conflictofinterest": {
+		id: "conflictofinterest",
+		name: "Conflict of Interest",
+		//Since levitate cant be coded in
+		onStart: function (pokemon) {
+			this.useMove('magnetrise', pokemon);
+		},
+		//Filter ghost and dark type attacks
+		onBasePowerPriority: 7,
+		onSourceBasePower: function (basePower, attacker, defender, move) {
```
Comment (DeathlyPlays) [inline mods/essb/abilities.js]: Simplify the ghost and dark both in this line

File: mods/essb/abilities.js [hunk 2]
```diff
@@ -698,4 +698,37 @@ exports.BattleAbilities = {
...
+			if (move.type === 'Ghost') {
+				return this.chainModify(0.5);
+			}
+		},
+		//HeatProof
+		onBasePowerPriority: 7,
+		onSourceBasePower: function (basePower, attacker, defender, move) {
+			if (move.type === 'Fire') {
```
Comment (DeathlyPlays) [inline mods/essb/abilities.js]: Same with the Fire type

File: mods/essb/moves.js [hunk 3]
```diff
@@ -1239,4 +1239,34 @@ exports.BattleMovedex = {
...
+			onAfterMoveSecondarySelf: function (pokemon, target, move) {
+				if (target.clearBoosts) this.boost({def: 1, spd: 1, spe: -1}, pokemon, pokemon, move);
+			},
+		category: "Physical",
+		name: "Moment of Inertia",
+		id: "momentofinertia",
+		pp: 10,
+		priority: 0,
+		flags: {contact: 1, protect: 1, mirror: 1},
+		secondary: false,
```
Comment (DeathlyPlays) [inline mods/essb/moves.js]: Missing ending },

Comment (DeathlyPlays) [reply mods/essb/moves.js]: Nvm Im on mobile mb

File: mods/essb/scripts.js [hunk 4]
```diff
@@ -508,6 +508,21 @@ exports.BattleScripts = {
 				},
 				nature: "Adamant",
 			},
+			"Almighty Bronzong": {
+				species: "Bronzong",
+				ability: "Conflict of Interest",
+				item: "Leftovers",
+				moves: ['recover', 'toxick', 'earthquake'],
```
Comment (DeathlyPlays) [inline mods/essb/scripts.js]: >Toxick

File: mods/essb/scripts.js [hunk 5]
```diff
@@ -508,6 +508,21 @@ exports.BattleScripts = {
...
+				gender: "N",
```
Comment (DeathlyPlays) [inline mods/essb/scripts.js]: Gender N?

**Review by DeathlyPlays — CHANGES_REQUESTED** [review]

File: mods/essb/scripts.js (hunk 5 above)
Comment (ghost) [reply mods/essb/scripts.js]: Neutral/Genderless

**Comment by ghost** [comment]
bar hex.js Travis Passed

**Pull request merged**